contracts/mocks/docs/utilities/Base64NFT.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {ERC721} from "@openzeppelin/contracts/token/ERC721/ERC721.sol";
```

Comment:
We should not need the `@openzeppelin/contracts`. The `../../..` are supposed to be replaced automatically during doc generation